### DISCUSSION TITLE
Rotate particle emitters

### DISCUSSION THREAD

**Discussion by SpieleFresser** [matteobruni/tsparticles#1693] [Q&A] [answered]
Hello

Sorry if this question is dumb but I didn't find a way of rotating the particle emitters. Is there a way to do this or is it not implemented?
I want to make one emitter for the topside of a skewed rectangle and one for the up-facing skewed side but for this to look good I need to rotate the second emitter.
I tried adding "rotation: 45" and "transform: {rotation: 45}" but neither of that worked.

Code:
https://codepen.io/spielefresser/pen/JjWgPwb

I hope you can help me, stay safe, and have a great day ^^

**Comment by matteobruni** [comment, marked as answer]
Hi @SpieleFresser 

I didn't understand well what do you mean with roating particles emitters. Do you want two emitters with a different angles like this sample? https://codepen.io/matteobruni/pen/qBqGPgQ
This can be achieved by changing the `direction` property of emitters, you can specify an angle in degrees or values like `top`, `top-right`, `right`, and so on.

Or an emitter that emits particles with a rotating angle everytime like a spiral? This is not actually supported.

**Reply by SpieleFresser** [reply]
Thanks for your response

I want the emitter itself to be rotated. In your example that wouldn't change anything because the emitters are points. I want my emitters to be on the two up-facing outlines of the skewed rectangle shape. and therefore I need to rotate the smaller one next to the skewed edge by 45°.
I hope that's a better explanation than before.

Stay safe, and have a great day ^^

**Reply by matteobruni** [reply]
This is the closest sample: https://codepen.io/matteobruni/pen/zYZgoxQ

The width and height of the emitter are dimensions of a square. Maybe in a future release I could implement the circle and a rotation of that shape.

**Reply by SpieleFresser** [reply]
Ok, still thank you for your time ^^

**Comment by atazmin** [comment]
Here is hat worked for me, setting "offset: 45"
...
move: {
            angle: { offset: 45, value: 5 },
            direction: "bottom",
            drift: 0,
            enable: true,
            random: false,
            size: false,
            speed: 8,           
            vibrate: false,
            warp: false,
          },

...
